_posts/2019-03-14-jump_dev_2019.md

```diff
@@ -197,3 +197,19 @@ Ver la [versión en inglés](#schedule).
 - Diego Moran (Universidad Adolfo Ibáñez; Organización Local)
 
 Contactar a jump-dev-committee at googlegroups.com para más información.
+
+[joaquim_dias_garcia]: https://drive.google.com/file/d/1XKQQTPb5iHNYgwyU9mlrHzvq6kY8Hw7X/view?usp=sharing
```

Comment:
The drive links to notebooks don't seem to work. We may have to commit those to the repository.